stack.yaml

```diff
@@ -37,10 +37,14 @@ packages:
 # Dependency packages to be pulled from upstream that are not in the resolver
 # using the same syntax as the packages field.
 # (e.g., acme-missiles-0.3)
-# extra-deps: []
+extra-deps:
+- llvm-hs-8.0.0
+- llvm-hs-pure-8.0.0
 
 # Override default flag values for local packages and extra-deps
-# flags: {}
+flags:
+  llvm-hs:
+    shared-llvm: false
```

Comment:
Maybe I should remove `llvm-hs` part.
Compiling LLVM is not super easy, so it is not good choice to enforce someone who wants to contribute to the front part of the compiler to download and compile LLVM, IMO.
I consider creating separated package for real code generation (i.e. for machine code generation) instead of using miniCUTE package itself.

Reply:
FYI, compiling LLVM in my low-end machine takes about 2h.

Reply:
Even if I divided packages, I still need to specify `llvm-hs` here, because this is one stack project.